patching/plugin-commands-patching/test/patch.test.ts

```diff
@@ -142,6 +142,44 @@ describe('patch and commit', () => {
     expect(fs.existsSync('node_modules/is-positive/license')).toBe(false)
   })
 
+  test('patch-commit with relative path', async () => {
```

Comment:
why is this test needed? It doesn't set a custom editDir

Reply:
The point of this test is in line 168

```typescript
    }, [path.relative(process.cwd(), patchDir)])
```

It does need custom edit dir to pass relative path to `patch-commit`.

I have made sure that this test fails with old implementation before committing it.